docs/en/operation/Journal.md

```diff
@@ -93,3 +94,49 @@ If the restore succeeds, you should see a log message along the lines of
 INFO AlluxioMasterProcess - Restored 57 entries from backup
 ```
 in the primary master logs.
+
+### Changing Masters
+
+If the journal is stored in a distributed system like HDFS, changing masters is easy.
+As long as the new master sets `alluxio.master.journal.folder` the same as the old
+master, it will start up in the same state that the old master left off.
+
+If the journal is stored on the local filesystem, the journal folder needs to be
+copied to the new master.
+
+## Advanced
+
+### Journal Checkpoints
+
+When running with a single master, the journal folder size will grow indefinitely
+as metadata operations are written to journal log files. To address this, production
+deployments should run in HA mode with multiple Alluxio masters. The non-primary
+masters will create checkpoints of the master state and clean up the logs that
+were written prior to the checkpoints. For example, if 3 million files were
```

Comment:
`3 million files`  , alluxio files right? let's be accurate to avoid confusion for local log files or block files.

Reply:
updated